server/util/url/url_test.go

```diff
@@ -0,0 +1,64 @@
+package url_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/buildbuddy-io/buildbuddy/server/config"
+	"github.com/buildbuddy-io/buildbuddy/server/environment"
+	"github.com/buildbuddy-io/buildbuddy/server/real_environment"
+	"github.com/buildbuddy-io/buildbuddy/server/util/healthcheck"
+	"github.com/buildbuddy-io/buildbuddy/server/util/url"
+)
+
+func envWithAppURL(t *testing.T, appUrl string) environment.Env {
+	c, err := config.NewConfiguratorFromData([]byte(fmt.Sprintf("app:\n  build_buddy_url: %s\n", appUrl)))
```

Comment:
nice, I like this new pattern for getting a minimal testenv :+1: will start using this in future tests

Reply:
@tempoz is going to fix everything™ and make all of these config vars flags soon which will make this much easier -- you just set the flag value for the lifespan of the test.